gamemode/core/meta/sh_item.lua

```diff
@@ -86,7 +107,8 @@ function ITEM:Print(detail)
 	end
 end
 
--- Dev Buddy, You don't have to make another function to print the item Data.
+--- A utility function printing all information about the item.
+-- @realm shared
 function ITEM:PrintData()
```

Comment:
Should be more clear that this method prints specifically the item's data

Reply:
Also done